Makefile

```diff
@@ -199,9 +199,14 @@ endif
 
 CRD_VERSIONS=v1
 CRD_OUTPUTS=build/crds
-DEVICES_VERSION=v1alpha2
+DEVICES_VERSION=v1beta1
+OPERATIONS_VERSION=v1alpha1
```

Comment:
This pr mainly fix the error caused by updating device version to v1beta1?  Please provide detail description in this PR.

Reply:
it would be also nice to update the doc section below accordingly.

Reply:
I'm having some issues with the test, so let's ignore it for now。